packages/unenv-preset/package.json

```
@@ -0,0 +1,61 @@
+{
+	"name": "@cloudflare/unenv-preset",
+	"version": "0.0.1",
+	"description": "cloudflare preset for unenv",
+	"keywords": [
+		"cloudflare",
+		"workers",
+		"cloudflare workers",
+		"Node.js",
+		"unenv",
+		"polyfills"
+	],
+	"homepage": "https://github.com/cloudflare/workers-sdk#readme",
+	"bugs": {
+		"url": "https://github.com/cloudflare/workers-sdk/issues"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/cloudflare/workers-sdk.git",
+		"directory": "packages/unenv-preset"
+	},
+	"license": "MIT OR Apache-2.0",
+	"sideEffects": false,
+	"exports": {
+		".": {
+			"types": "./dist/index.d.ts",
+			"require": "./dist/index.cjs",
+			"import": "./dist/index.mjs"
+		},
+		"./runtime/*": {
+			"types": "./dist/runtime/*.d.ts",
+			"require": "./dist/runtime/*.cjs",
+			"import": "./dist/runtime/*.mjs"
+		}
+	},
+	"main": "./dist/index.cjs",
+	"types": "./dist/index.d.ts",
+	"files": [
+		"dist",
+		"runtime"
+	],
+	"scripts": {
+		"build": "unbuild",
+		"check:lint": "eslint",
+		"check:type": "tsc --noEmit"
+	},
+	"devDependencies": {
+		"@types/node": "^22.9.3",
+		"typescript": "catalog:default",
+		"unbuild": "^2.0.0",
+		"wrangler": "3.95.0"
+	},
+	"peerDependencies": {
+		"workerd": "^1.20241205.0"
+	},
+	"peerDependenciesMeta": {
+		"workerd": {
+			"optional": true
+		}
+	}
```

If you add this here:

```suggestion
	},
	"workers-sdk": {
		"prerelease": true
	}
```

Then we should auto-generate pre-release builds per PR.